build.sh

```diff
@@ -295,6 +296,10 @@ while :; do
         verbose)
             __VerboseBuild=1
             ;;
+	staticcurl)
+            staticCurl=" -DCMAKE_STATIC_CURL=1"
+            __CMakeExtraArgs="$__CMakeExtraArgs$staticCurl"
```

Comment:
NIT: could this just be `__CMakeExtraArgs="$__CMakeExtraArgs -DCMAKE_STATIC_CURL"` and remove setting `staticCurl` above?